authorizer-app/src/app/components/rest-source-authorization/update-rest-source-user.component.ts

```diff
@@ -83,19 +106,55 @@ export class UpdateRestSourceUserComponent implements OnInit {
     );
   }
 
-  private cancelUpdateUser() {
-    return this.router.navigate(['/users']);
-  }
-
   private addRestSourceUser(code: string, state: string) {
     this.restSourceUserService.addAuthorizedUser(code, state).subscribe(
       data => {
+        this.onChangeProject(data.projectId)
         this.restSourceUser = data;
       },
       (err: Response) => {
+        console.log('Cannot retrieve current user details', err)
         this.errorMessage = 'Cannot retrieve current user details';
         window.setTimeout(() => this.router.navigate(['']), 5000);
       }
     );
   }
+
+  private loadAllRestSourceProjects() {
+    this.restSourceUserService.getAllProjects().subscribe(
+      (data: any) => {
+        this.restSourceProjects = data.projects;
+        this.initialize()
+      },
+      () => {
+        this.errorMessage = 'Cannot load projects!';
+      }
+    );
+  }
+
+  onChangeProject(projectId: string) {
+    if(projectId === ''){
+    }else{
+      this.loadAllRestSourceUsersOfProject(projectId)
+    }
+  }
+
+  private loadAllRestSourceUsersOfProject(projectId: string) {
+    this.restSourceUserService.getAllUsersOfProject(projectId).subscribe(
```

Comment:
This call is till wrong. Here we should retrieve '/projects/{projectId}/users'.
I will make the changes and commit.

Reply:
Thanks. This is the endpoint which I got confused.